core/object/object.h

```diff
@@ -769,7 +769,6 @@ class Object {
 	static void initialize_class();
 	_FORCE_INLINE_ static void register_custom_data_to_otdb() {}
 
-public:
```

Comment:
I'd keep this as the above one has a note and might be changed in the future, and could then cause mistakes